### PR TITLE
Feature/MOB-67-header

### PR DESCRIPTION
**Pull request by MPilinkus**

### CODE REVIEW

File: src/app/landing-page/landing-section/landing-section.component.scss [hunk 1]
```diff
@@ -52,6 +53,7 @@
 
 @media only screen and (max-width: 1200px) {
   .landing-section {
+    box-shadow: inset 0px 10px 10px -5px rgba(0,0,0,0.25);
```
Comment (lukasrakauskas) [inline src/app/landing-page/landing-section/landing-section.component.scss]: Duplicate unnecessary?

File: src/app/shared/button/button.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {Constants} from '../constants';
```
Comment (lukasrakauskas) [inline src/app/shared/button/button.component.ts]: Consistency please with spaces around imports

File: src/app/shared/button/button.component.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+})
+export class ButtonComponent {
+
+  @Input() buttonStyle: string;
+  @Input() isBig: boolean;
+  @Output() buttonClicked: EventEmitter<any> = new EventEmitter<any>();
+
+  secondStyle = Constants.Button.Secondary;
+  tertiaryStyle = Constants.Button.Tertiary;
+
```
Comment (wizacass) [inline src/app/shared/button/button.component.ts:17]: Is this empty line required?

**Comment by wizacass** [comment]
Next time please write commit messages as described in our wiki: https://github.com/KamandaX/Wiki/blob/master/Development_Rules.md

> All commits should start with a verb in imperative form

File: src/app/shared/button/button.component.ts (hunk 3 above)
Comment (MPilinkus) [reply src/app/shared/button/button.component.ts:17]: No, in theory you can write all of the code in one line, but extra lines makes it more readable, don't they?

Comment (wizacass) [reply src/app/shared/button/button.component.ts:17]: Fair enough.

**Review by wizacass — APPROVED** [review]

**Review by lukasrakauskas — APPROVED** [review]

**Pull request merged**